### ISSUE TITLE
No sound in iOS 15

### ISSUE THREAD

**Issue by guillaumebrunerie** [pixijs/sound#274]
My app works fine on Android but there is no sound at all in Safari on iOS 15. The README says that iOS 11+ is supposed to be supported. The sounds on https://pixijs.io/sound/examples/ don't work either.

Is this a known issue? I tried to set `useLegacy` to `true` but it didn't make any difference.

Not sure how to get some console logs from iOS.

**Comment by braveltd** [comment]
Make sure you use .mp3 and not .ogg or .wav

**Comment by guillaumebrunerie** [comment]
Yes, I do use `.mp3`, and so does https://pixijs.io/sound/examples/, but it does not produce any sounds.

**Comment by braveltd** [comment]
I have ios 18 and it works. is it only on iOS 15 or from iOS 15?

**Comment by guillaumebrunerie** [comment]
Hmm, I don’t have an iOS 18 device to test, but I tested on two different iOS 15 devices and didn’t have sound in either. Might be an issue specific to iOS 15.

**Comment by gpoole** [comment]
@guillaumebrunerie I also had issues with no sound and found that in Safari on iOS [the ringer switch being off mutes WebAudioContext but not other kinds of web audio](https://github.com/pixijs/sound/issues/251#issuecomment-2667276569). You might want to check if that's the case for you too?
